src/rtos/nuttx.c

```diff
@@ -287,16 +314,24 @@ static int nuttx_update_threads(struct rtos *rtos)
 			thread = &rtos->thread_details[thread_count - 1];
 			thread->threadid = tcb_addr;
 			thread->exists = true;
+			thread->extra_info_str = malloc(256);
+			snprintf(thread->extra_info_str, 256, "pid:%d",
+				tcb.dat[pid_offset - 8] |
+				tcb.dat[pid_offset - 8 + 1] << 8);
 
```

Comment:
Thanks for re-adding pid info which I removed by mistake when I merged v0.10.0, but could you create a separate commit in this PR? Because these changes does not relate to FPU support.